openmls/src/prelude_test.rs

```diff
@@ -14,4 +14,4 @@ pub use crate::group::tests::{
 };
 pub use crate::schedule::kat_key_schedule::{self, KeyScheduleTestVector};
 pub use crate::tree::tests_and_kats::kats::kat_encryption::{self, EncryptionTestVector};
-pub use crate::treesync::tests_and_kats::kats::kat_tree_kem::{self, TreeKemTestVector};
+// pub use crate::treesync::tests_and_kats::kats::kat_tree_kem::{self, TreeKemTestVector};
```

Comment:
If we don't need it, we don't need it.

Reply:
We need it. But it doesn't work. I added the issue to make this work again.